Keas.Mvc/Models/HomeViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using Keas.Core.Data;
+using Keas.Core.Domain;
+using System.Linq;
+using System.Threading.Tasks;
+using Keas.Mvc.Services;
+using Microsoft.EntityFrameworkCore;
+
+namespace Keas.Mvc.Models
+{
+    public class HomeViewModel
+    {
+        public bool PendingItems { get; set; }
+
+
+
+        public static async Task<HomeViewModel> Create(ApplicationDbContext context, Person person)
+        {   
+            var viewModel = new HomeViewModel();
```

Comment:
I don't love doing 3 queries on the homepage but at least it's pretty simple.  Nothing to worry about but let's keep an eye on the performance and make sure it doesn't tax the azure db

Reply:
Yeah...I wasn't super thrilled about putting this here. My assumption is at a future date we do this once for a user and set some session flag that gets checked every 5 minutes or so in Ver 2.0.